visearch-android/src/main/java/com/visenze/visearch/android/ProductBaseSearchParams.java

```diff
@@ -0,0 +1,470 @@
+    public String getVttSource() {
+        return vttSource;
+    }
+
+    public void setVttSource(String vttSource) {
+        this.vttSource = vttSource;
+    }
+
+    public String getVaSdk() {
+        return vaSdk;
+    }
+
+    public void setVaSdk(String vaSdk) {
+        this.vaSdk = vaSdk;
+    }
+
+    public String getVaSdkVersion() {
+        return vaSdkVersion;
+    }
+
+    public void setVaSdkVersion(String vaSdkVersion) {
+        this.vaSdkVersion = vaSdkVersion;
+    }
+
+    public String getVaOs() {
+        return vaOs;
+    }
+
+    public void setVaOs(String vaOs) {
+        this.vaOs = vaOs;
+    }
+
+    public String getVaOsv() {
+        return vaOsv;
+    }
+
+    public void setVaOsv(String vaOsv) {
+        this.vaOsv = vaOsv;
+    }
+
+    public String getVaDeviceBrand() {
+        return vaDeviceBrand;
+    }
+
+    public void setVaDeviceBrand(String vaDeviceBrand) {
+        this.vaDeviceBrand = vaDeviceBrand;
+    }
+
+    public String getVaDeviceModel() {
+        return vaDeviceModel;
+    }
+
+    public void setVaDeviceModel(String vaDeviceModel) {
+        this.vaDeviceModel = vaDeviceModel;
+    }
+
+    public String getVaAppBundleId() {
+        return vaAppBundleId;
+    }
+
+    public void setVaAppBundleId(String vaAppBundleId) {
+        this.vaAppBundleId = vaAppBundleId;
+    }
+
+    public String getVaAppName() {
+        return vaAppName;
+    }
+
+    public void setVaAppName(String vaAppName) {
+        this.vaAppName = vaAppName;
+    }
+
+    public String getVaAppVersion() {
+        return vaAppVersion;
+    }
+
+    public void setVaAppVersion(String vaAppVersion) {
+        this.vaAppVersion = vaAppVersion;
+    }
+
+    public String getVaAaid() {
+        return vaAaid;
+    }
+
+    public void setVaAaid(String vaAaid) {
+        this.vaAaid = vaAaid;
+    }
+
+    public String getVaDidmd5() {
+        return vaDidmd5;
+    }
+
+    public void setVaDidmd5(String vaDidmd5) {
+        this.vaDidmd5 = vaDidmd5;
+    }
+
+    public String getVaN1() {
+        return vaN1;
+    }
+
+    public void setVaN1(String vaN1) {
+        this.vaN1 = vaN1;
+    }
+
+    public String getVaN2() {
+        return vaN2;
+    }
+
+    public void setVaN2(String vaN2) {
+        this.vaN2 = vaN2;
+    }
+
+    public String getVaS1() {
+        return vaS1;
+    }
+
+    public void setVaS1(String vaS1) {
+        this.vaS1 = vaS1;
+    }
+
+    public String getVaS2() {
+        return vaS2;
+    }
+
+    public void setVaS2(String vaS2) {
+        this.vaS2 = vaS2;
+    }
+
+
+    @SerializedName("filters")
+    private List<String> filters;
+
+    @SerializedName("text_filters")
+    private List<String> textFilters;
+
+    @SerializedName("attrs_to_get")
+    private List<String> attrsToGet;
+
+    @SerializedName("facets")
+    private List<String> facets;
+
+
+    @SerializedName("custom_map")
+    @Expose(deserialize = false, serialize = false)
+    private Map<String, String> customMap;
```

Comment:
rename to customParams

Reply:
why do we need a "SerializedName" ? can remove?

Reply:
changed already